tests/test_webhook.py

```diff
@@ -39,4 +39,4 @@ def test_webhooks(per_run_unique):
         easypost.Webhook.retrieve(webhook.id)
 
     assert exception_context.value.http_status == 404
-    assert exception_context.value.message == "The requested resource could not be found."
+    assert exception_context.value.message.get("message") == "The requested resource could not be found."
```

Comment:
It's a code smell when we have to adjust tests to fit changed behavior. Tests are typically the "standard" the code should live by and when assertions change, it's an indicator that behavior is changing for our users too. For instance, users may be relying on the fact we unfurl the `message` key - with this change, we may inadvertently be introducing changed behavior around error handling which could break integrations.

I'm unsure this is the proper change here.

Reply:
I think you're right, another way to fix this is bypassing **http_body** instead of **message** in the Error class, so the Exception is going to print out the entire error message from API.